benchmarks/plot.jl

```diff
@@ -28,10 +28,16 @@ of points and plot the results.
 - `perturbation_factor_position = 1.0`: Perturb point positions by this factor. A factor of
                                         `1.0` corresponds to points being moved by
                                         a maximum distance of `0.5` along each axis.
+
+# Example
+
+also an example?
 """
 function plot_benchmarks(benchmark, n_points_per_dimension, iterations;
                          parallel = true, title = "",
                          seed = 1, perturbation_factor_position = 1.0)
+    # Is there a reason why this is hard coded? Why not passing custom nhs like `custom_nhs...`?
+    # And then `nameof(typeof(nhs))`
```

Comment:
1. You would have to pass an array of functions `(search_radius, coordinates) -> nhs`, which would get ugly quickly.
2. `nameof(typeof(nhs))` is not enough. With new backends and options, many of these names will be "`SomeNeighborhoodSearch` with `some_option=true`".

I figured it is easier to treat this like an example file that you modify and include again.